test/acceptance/tests/consul-dns/consul_dns_test.go

```diff
@@ -0,0 +1,80 @@
+package consuldns
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const PODNAME = "dns-pod"
```

Comment:
Let's use camelcase for constants. I think this is the convention in golang.
```suggestion
const podName = "dns-pod"
```